<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="lathabp-ThinkPad-L470-W10DG" ignored="0" name="Default test" tests="1" failures="1" timestamp="17 Dec 2019 11:22:56 GMT" time="23.555" errors="0">
  <testcase name="ValideLogout" time="16.507" classname="test.logoutTest">
    <failure type="java.lang.AssertionError" message="expected [babloo] but found [[[[io.appium.java_client.android.AndroidDriver, Capabilities: {appActivity=com.lulu.commerce.SplashActivity, appPackage=com.lulu.commerce, databaseEnabled=false, desired={appActivity=com.lulu.commerce.SplashActivity, appPackage=com.lulu.commerce, noReset=true, platformVersion=10, platformName=android, deviceName=Pixel 2, fullReset=false}, deviceManufacturer=Google, deviceModel=Pixel 2, deviceName=FA7AC1A07832, deviceScreenSize=1080x1920, deviceUDID=FA7AC1A07832, fullReset=false, javascriptEnabled=true, locationContextEnabled=false, networkConnectionEnabled=true, noReset=true, platform=LINUX, platformName=Android, platformVersion=10, takesScreenshot=true, warnings={}, webStorageEnabled=false}] -&amp;gt; id: com.lulu.commerce:id/etxtName]]]">
      <![CDATA[java.lang.AssertionError: expected [babloo] but found [[[[io.appium.java_client.android.AndroidDriver, Capabilities: {appActivity=com.lulu.commerce.SplashActivity, appPackage=com.lulu.commerce, databaseEnabled=false, desired={appActivity=com.lulu.commerce.SplashActivity, appPackage=com.lulu.commerce, noReset=true, platformVersion=10, platformName=android, deviceName=Pixel 2, fullReset=false}, deviceManufacturer=Google, deviceModel=Pixel 2, deviceName=FA7AC1A07832, deviceScreenSize=1080x1920, deviceUDID=FA7AC1A07832, fullReset=false, javascriptEnabled=true, locationContextEnabled=false, networkConnectionEnabled=true, noReset=true, platform=LINUX, platformName=Android, platformVersion=10, takesScreenshot=true, warnings={}, webStorageEnabled=false}] -> id: com.lulu.commerce:id/etxtName]]]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at feature.logoutFeature.gotoLogoutUser(logoutFeature.java:18)
	at test.logoutTest.ValideLogout(logoutTest.java:10)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ValideLogout -->
</testsuite> <!-- Default test -->
